src/entity.ts

```diff
@@ -667,6 +683,15 @@ export namespace entity {
     throw new Error('Unsupported field value, ' + value + ', was provided.');
   }
 
+  const warningTypesIssued = new Set<string>();
+  const warn = (warningName: string, warningMessage: string) => {
+    console.log(warningName);
```

Comment:
It feels a bit odd to me to both `console.log` and `process.emitWarning`, as the default warning handler prints the error too. If we're keeping both, `console.warn` seems like the thing to do here. (Or is this a debug line that was left in?)

Reply:
'doh'